packages/workspace/src/utilities/cli-config-utils.ts

```diff
@@ -0,0 +1,5 @@
+import { NxJsonConfiguration, readJson, Tree } from '@nrwl/devkit';
+
+export function getNpmScope(host: Tree): string {
```

Comment:
This one shouldn't be needed, we can use the `getWorkspaceLayout` from the Nx DevKit.

Reply:
Awesome!